.github/workflows/release.yml

```diff
@@ -0,0 +1,113 @@
+name: release
+
+on:
+  workflow_dispatch:
+    inputs:
+      publishPreRelease:
```

Comment:
We don't need the `publishPreRelease` entry since this isn't doing pre-releases currently. Just make sure to remove the corresponding references in the logic.